crates/types/src/config/mod.rs

```diff
@@ -80,6 +80,16 @@ pub fn data_dir(dir: &str) -> PathBuf {
     guard.path().join(dir)
 }
 
+#[cfg(not(any(test, feature = "test-util")))]
+pub fn node_filepath(filename: &str) -> PathBuf {
+    data_dir(filename)
+}
+
+#[cfg(any(test, feature = "test-util"))]
+pub fn node_filepath(filename: &str) -> PathBuf {
+    data_dir(filename)
+}
+
```

Comment:
Are these identical?

Reply:
Yeah, doesn't make sense. Will remove it.